misc/userscripts/README.md

```diff
@@ -53,9 +53,10 @@ The following userscripts can be found on their own repositories.
 - [qtb.us](https://github.com/Chinggis6/qtb.us): small pack of userscripts.
 - [pinboard.zsh](https://github.com/dmix/pinboard.zsh): Add URL to your
   [Pinboard][] bookmark manager.
+- [qute-capture](https://github.com/alcah/qute-capture): Capture links with org
```

Comment:
This probably reads weird to anyone who doesn't know emacs. Maybe "with org" -> "with emacs' org-mode"?